pylossless/pipeline.py

```diff
@@ -768,21 +777,20 @@ def get_epochs(self, detrend=None, preload=True):
             an instance of mne.Epochs
         """
         # TODO: automatically load detrend/preload description from MNE.
-        tmin = self.config['epoching']['epochs_args']['tmin']
-        tmax = self.config['epoching']['epochs_args']['tmax']
-        overlap = self.config['epoching']['overlap']
-        events = mne.make_fixed_length_events(self.raw, duration=tmax-tmin,
-                                              overlap=overlap)
-
+        logger.info("🧹 Epoching..")
+        events = self.get_events()
         epoching_kwargs = self.config['epoching']['epochs_args']
         if detrend is not None:
             epoching_kwargs['detrend'] = detrend
         epochs = mne.Epochs(self.raw, events=events,
                             preload=preload, **epoching_kwargs)
-        epochs = (epochs.pick(picks=None, exclude='bads')
+        epochs = (epochs.pick(picks=picks, exclude='bads')
```

Comment:
picks is passed into the function and right now is to assure that we are always only picking EEG channels.